tsweb/tsweb.go

```diff
@@ -42,6 +42,7 @@ func NewMux(debugHandler http.Handler) *http.ServeMux {
 
 func registerCommonDebug(mux *http.ServeMux) {
 	expvar.Publish("counter_uptime_sec", expvar.Func(func() interface{} { return int64(Uptime().Seconds()) }))
+	expvar.Publish("num_goroutines", expvar.Func(func() interface{} { return runtime.NumGoroutine() }))
```

Comment:
can probably drop the `num_` prefix. Then you need a type annotation for it to make it through tsweb's VarzHandler else looks like you'll hit:

```
# skipping expvar %q (Go type %T%s) with undeclared Prometheus type
```

Reply:
Oops. Done, thanks.